modules/blockuser.cpp

```diff
@@ -175,7 +180,7 @@ class CBlockUser : public CModule {
 		vector<CClient*> vpClients = pUser->GetAllClients();
 		vector<CClient*>::iterator it;
 		for (it = vpClients.begin(); it != vpClients.end(); ++it) {
-			(*it)->PutStatusNotice(MESSAGE);
+			(*it)->PutStatusNotice(sMessage.empty() ? DEFAULT_MESSAGE : sMessage);
```

Comment:
Like the other code suggestion I just made...

```(*it)->PutStatusNotice(Blocked: " + sMessage.empty() ? DEFAULT_MESSAGE : sMessage));``` would actually disconnect the user with a "Blocked: REASON" message, like "Blocked: Your account has been disabled. Contact your administrator." for the default.  Like the other suggestion, it works better this way with Custom ban reasons, but also fairly decently for the default message.

Again, just my two cents.